corehq/apps/app_manager/views/forms.py

```diff
@@ -722,7 +726,9 @@ def commtrack_programs():
             "app_manager/v1/form_view.html",
             "app_manager/v2/form_view.html",
         )
-        return template, context
+
+    context.update({'case_config_options': case_config_options})
```

Comment:
OOC, why is this way down at the bottom of the function rather than in the initial dict, near where it's defined?  Also, this function looks like a great candidate for the next refactorathon